subprojects/docs/src/docs/release/notes.md

```diff
@@ -18,6 +18,21 @@ The new metadata format is still under active development, but it can already be
 ### Example new and noteworthy
 -->
 
+### Specifying metadata sources for repositories
+
+Gradle now allows you to explicitly state for [which metadata files](userguide/repository_types.html#sub:supported_metadata_sources) it should search in a repository. You can use this, for example, to gain efficiency when querying a Maven repository by telling Gradle not to look for plain `jar` files anymore when the `pom` file for a module is missing.
+
+    repositories {
+         mavenCentral {
+             metadataSources {
+                 mavenPom() // Look for Maven '.pom' files
+                 // artifact() - Do not look for artifacts without metadata file
+             }
+         }
+    }
+
+You can also use this to selectively opt into using the new Gradle metadata format for one repository using `metadataSources { gradleMetadata() }`.
```

Comment:
:o: "selectively opt into" => "programmatically opt-in to"

Reply:
:x: Also, is this the _first ever_ public mention of the new Gradle metadata format? I can already read the minds of readers saying "Wait... what?", like Anna from Frozen does, when they read this. 

Perhaps we should either omit this or highlight it with a different section, otherwise users won't understand why they'd want to opt-in to this thing.

Reply:
I agree that we have to be careful here. The metadata format is introduced in the release notes in the paragraph right above this one. Please have a look at that paragraph as well and tell me what you think. (I am afraid no one actually reviewed that yet.)